contributed_definitions/nyaml/NXapm_compositionspace_config.yaml

```diff
@@ -1,25 +1,12 @@
 category: application
 doc: |
-  Config for a run with Alaukik Saxena's composition space tool.
+  Application definition for a configuration of the CompositionSpace tool used in atom probe.
   
-  This is an initial draft application definition for the common NFDI-MatWerk,
-  FAIRmat infrastructure use case IUC09 how to improve the organization and
-  results storage of the composition space tool and make these data at the
-  same time directly understandable for research data management systems
-  like NOMAD Oasis.
-symbols:
-  doc: |
-    The symbols used in the schema to specify e.g. dimensions of arrays.
-  n_voxel: |
-    Number of voxel of discretized domain for analyzed part of the dataset.
-  d: |
-    The dimensionality of the grid.
-  c: |
-    The cardinality or total number of cells/grid points.
-  n_clst_dict: |
-    Number of terms in the composition clustering dictionary
-  n_spat_dict: |
-    Number of terms in the position clustering dictionary
+  * `A. Saxena et al. <https://www.github.com/eisenforschung/CompositionSpacetool.git>`_
```

Comment:
Link seems to be broken, I guess it's https://github.com/eisenforschung/CompositionSpace?

Reply:
Thank you that was a mistake indeed